File: app/src/main/java/com/theflexproject/thunder/model/Crew.java
package com.theflexproject.thunder.model;

// Model untuk Crew
public class Crew implements MyMedia {
    private int id;
    private String name;
    private String job;
    private String profilePath;
    private String department;

    public Crew(int id, String name, String job, String department, String profilePath) {
        this.id = id;
        this.name = name;
        this.job = job;
        this.department = department;
        this.profilePath = profilePath;
    }

    // Getters
    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getJob() {
        return job;
    }
    public String getDepartment() {
        return department;
    }

    public String getProfilePath() {
        return profilePath;
    }
}

File: app/src/main/java/com/theflexproject/thunder/model/Crew.kt
package com.theflexproject.thunder.model

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize

@JsonClass(generateAdapter = true)
@Parcelize
data class Crew(
    @Json(name = "id") val id: Int = 0,
    @Json(name = "name") val name: String? = null,
    @Json(name = "job") val job: String? = null,
    @Json(name = "department") val department: String? = null,
    @Json(name = "profile_path") val profilePath: String? = null
) : MyMedia, Parcelable
